ui/src/app/machines/views/MachineDetails/MachineStorage/StorageNotifications/StorageNotifications.tsx

```diff
@@ -38,8 +38,8 @@ const StorageNotifications = ({ id }: Props): JSX.Element | null => {
             active: true,
             content:
               "No storage information. Commission this machine to gather storage information.",
-            status: "Error:",
-            type: "negative" as const,
+            severity: "negative" as const,
```

Comment:
I kind of wonder if it would be less strange to just use the enum for these.

Reply:
Yeah probably! :+1: